docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -18,13 +18,13 @@ A device can have various physical and virtual sensors. The following table list
 
 | Supported sensor types                   |                                          |                                       |
 |----------------------------------------|----------------------------------------|-------------------------------------|
-| [Accelerometer](#accelerometer)          | [Heart Rate Monitor Sensor](#hrm)        | [Proximity Sensor](#proximity)        |
-| [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Humidity Sensor](#humidity)             | [Rotation Vector Sensor](#rotation)   |
-| [Gravity Sensor](#gravity)               | [Light Sensor](#light)                   | [Sleep Monitor](#sleep_monitor)       |
-| [Gyroscope](#gyro)                       | [Linear Acceleration Sensor](#lin_accelerometer) | [Temperature Sensor](#temperature)    |
-| [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Magnetic Sensor](#magnetic)             | [Ultraviolet Sensor](#ultraviolet)    |
-| [Pressure Sensor](#pressure)             | [Orientation Sensor](#orientation)       | [Uncalibrated Gyroscope](#uncal_gyro) |
-| [Pedometer](#pedometer)                  | [Uncalibrated Magnetic Sensor](#uncal_magnetic) | -                                     |
+| [Accelerometer](#accelerometer)  | [Humidity Sensor](#humidity)   | [Proximity Sensor](#proximity)   |
+| [AutoRotation Sensor](#auto_rotation)   | [Light Sensor](#light)   | [Rotation Vector Sensor](#rotation)  |
+| [Geomagnetic Rotation Vector Sensor](#mag_rotation)  | [Linear Acceleration Sensor](#lin_accelerometer)  | [Sleep Monitor](#sleep_monitor)  |
+| [Gravity Sensor](#gravity)   | [Magnetic Sensor](#magnetic) | [Temperature Sensor](#temperature)  |
+| [Gyroscope](#gyro)    | [Orientation Sensor](#orientation)  | [Ultraviolet Sensor](#ultraviolet)  |
+| [Gyroscope Rotation Vector Sensor](#gyro_rotation)  | [Pressure Sensor](#pressure)  | [Uncalibrated Gyroscope](#uncal_gyro)  |
+| [Heart Rate Monitor Sensor](#hrm)   | [Pedometer](#pedometer)   | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
```

Comment:
Thanks for the update.
However there was a mistake at previous comment. 
Would you please change the locations of "Pedometer" and "Pressure Sensor"?

![image](https://user-images.githubusercontent.com/9958374/82785885-6b461d80-9e9e-11ea-81b0-ee9a4a2ccb1e.png)

Reply:
@mijin85cho done :smile: